### PR TITLE
Add vulnerabilities section and note about manual edits to readme

### PR DESCRIPTION
**Pull request by MichaelSimons**

### CODE REVIEW

**Review by mthalman — APPROVED** [review]

**Comment by oleksandr-didyk** [comment]
Thanks for adding this

**Comment by oleksandr-didyk** [comment]
Aaaaaaaaaaaaaaand instead of approving I merged it. My bad

**Comment by MichaelSimons** [comment]
> Aaaaaaaaaaaaaaand instead of approving I merged it. My bad

No worries, you just eliminate a button push I needed to perform 😄

File: README.md [hunk 1]
```diff
@@ -74,6 +75,16 @@ targeting pack is needed, please [open a new issue](#filing-issues) to discuss.
 ./generate.sh --type text --package microsoft.build.traversal,3.1.6
 ```
 
+## Vulnerable Packages
+
+CVEs may exist for reference packages included in this repo. If they are mitigated by a newer version, the newer version should be added, the vulnerable version should be removed, and references to the vulnerable package within other reference
```
[the remark this answers is not in the document]
Comment (MichaelSimons) [reply README.md:80]: That is correct.  I could word this better.

**Pull request merged**